python/tvm/relax/transform/lazy_transform_params.py

```diff
@@ -225,10 +236,38 @@ class LazyTransformParams:
     (Load the input to memory on demand, and immediately free it after the last use.)
 
     Note: ToNonDataflow() and RemovePurityTracking() should be invoked before this pass.
+
+    Parameters
+    ----------
+    fget_item: str
+        The name of the get_item function.
+    fset_item: str
+        The name of the set_item function.
+    get_item_param: list of relax.Var
+        The parameters of the get_item function except index.
+        The given parameters will be placed before index.
+        For example, if get_item_param is [param1, param2], then the pass will generate
+        call_packed(fget_item, [param1, param2, index])
+    set_item_param: list of relax.Var
+        The parameters of the set_item function except index and value.
+        The given parameters will be placed before index and value.
+        For example, if set_item_param is [param1, param2], then the pass will generate
+        call_packed(fset_item, [param1, param2, index, value])
     """
 
+    def __init__(
+        self, fget_item="get_item", fset_item="set_item", get_item_param=[], set_item_param=[]
```

Comment:
Using a mutable object as a default value can have unexpected consequences.  (e.g. Accidental mutation of an object, which is reused in the next call.)  Instead of having a list as the default value, it is better to have `get_item_param=None`, then to construct the list within the body of the function.